samples/Sandbox/App.axaml.cs

```diff
@@ -16,7 +16,14 @@ public override void OnFrameworkInitializationCompleted()
             if (ApplicationLifetime is IClassicDesktopStyleApplicationLifetime desktopLifetime)
             {
                 desktopLifetime.MainWindow = new MainWindow();
+
+                desktopLifetime.ShutdownRequested += DesktopLifetime_ShutdownRequested;
             }
         }
+
+        private void DesktopLifetime_ShutdownRequested(object sender, System.ComponentModel.CancelEventArgs e)
```

Comment:
This should be removed before merging.

Reply:
removed... just wanted to show how it would be used.